compiler/il/OMRResolvedMethodSymbol.cpp

```diff
@@ -70,6 +70,7 @@
 #include "runtime/Runtime.hpp"
 #include "infra/SimpleRegex.hpp"
 
+#define OPT_DETAILS "O^O OSR: "
```

Comment:
We can reword this to be more generic since the message has enough detail to indicate the transformation is related to OSR call. So maybe reword to `IL Resolved Method Symbol`.